contributors/guide/help-wanted.md

```diff
@@ -49,6 +49,10 @@ Items marked with the `help wanted` label need to ensure that they are:
   Often these issues become obsolete and have already been done, are no longer
   desired, no longer make sense, have changed priority or difficulty , etc.
 
+Example of a Help Wanted issue:
+
+  [Add /remove-(approve|lgtm|hold) commands test-infra#21356 (comment)](https://github.com/kubernetes/test-infra/issues/21356#issuecomment-799972711)
+  
```

Comment:
Now that I think about it, saying something as the following might be more appropriate (mainly because the link is to a comment that describes the task that needs to be done and not an issue itself):

Example of a Help Wanted issue description can be found here: [kubernetes/test-infra#21356 (comment)](https://github.com/kubernetes/test-infra/issues/21356#issuecomment-799972711).

Reply:
Yeah you're right! Looks much more presentable. Alright I'll incorporate it this way. Thanks :)